content/pages/index.md

```diff
@@ -102,30 +102,26 @@ All the developer resources to help you get the most from Box products
       href="/reference/">
       *Box API Reference*
     </Tile>
-    <Tile type="python" title="Python Gen SDK"
-    href="https://github.com/box/box-python-sdk-gen?tab=readme-ov-file#box-python-sdk-generated">
-      *box-python-sdk-gen*
+    <Tile type="python" title="Python SDK"
+    href="https://github.com/box/box-python-sdk">
+      *box-python-sdk*
     </Tile>
     <Tile type="java" title="Box Java SDK"
     href="https://github.com/box/box-java-sdk#readme">
```

Comment:
remove #readme to be consistent (optionally)

Reply:
added to all